### PR TITLE
fix: Update random device profile, configuration and discovery device configuration file

### PR DESCRIPTION
**Pull request by nil4git**
Update discovery sample app configuration.toml to include devices resource directory as well which will assist in creation of discovery device/s

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-c/blob/master/.github/CONTRIBUTING.md

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

## New Imports

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by nil4git** [comment]
Hi Team,
Request to review the same
Thanks

**Review by iain-anderson — CHANGES_REQUESTED** [review]
Please add a devices directory so as to avoid the service emitting an error at startup.
git will not allow for empty directories so add a readme.txt stating that static device definitions may be added here

**Comment by iain-anderson** [comment]
If you could also add the fix for the profile in the device-random example that would be great

**Comment by nil4git** [comment]
> If you could also add the fix for the profile in the device-random example that would be great

Done please check

File: src/c/examples/random/res/configuration.toml [hunk 1]
```diff
@@ -10,6 +10,8 @@
   RequestTimeout = '5s'
   StartupMsg = 'Example random device started'
   HealthCheckInterval = '10s'
+  Host = ''	# Insert docker hosted ip address here
```
Comment (iain-anderson) [inline src/c/examples/random/res/configuration.toml:13]: Service will not start with Host set to an empty string, please remove this line

**Review by iain-anderson — CHANGES_REQUESTED** [review]

File: src/c/examples/random/res/configuration.toml (hunk 1 above)
Comment (nil4git) [reply src/c/examples/random/res/configuration.toml]: Thought of putting a comment to update the Host address as per the docker hosting
Removed by the way
Please check

**Review by iain-anderson — APPROVED** [review]
LGTM

**Pull request merged**